code/modules/mapfluff/ruins/objects_and_mobs/necropolis_gate.dm

```diff
@@ -50,7 +50,7 @@
 	)
 
 	AddElement(/datum/element/connect_loc, loc_connections)
-	AddComponent(/datum/component/seethrough, SEE_THROUGH_MAP_DEFAULT_TWO_TALL)
+	AddComponent(/datum/component/seethrough, SEE_THROUGH_MAP_DEFAULT_TWO_TALL, null, null, null, FALSE)
```

Comment:
```suggestion
	AddComponent(/datum/component/seethrough, SEE_THROUGH_MAP_DEFAULT_TWO_TALL, clickthrough = FALSE)
```
Don’t overwrite the default parameters with null or you’ll break a lot of stuff. Just use a named parameter as above